htdocs/challenges.php

```diff
@@ -130,6 +136,7 @@
 
     // if the challenge isn't available yet, display a message and continue to next challenge
     if ($time < $challenge['available_from']) {
+        if($challenge['expose'] ==1) {
```

Comment:
These two lines could be
``
if ($time < $challenge['available_from'] && $challenge['expose']) {
``